manuscript/3-SoftConstr.txt

```diff
@@ -18,49 +18,48 @@
 ## Εισαγωγή
 
 
+
 Ο όρος κατασκευή λογισμικού αναφέρεται στη λεπτομερή κατασκευή ενός
-λειτουργικού λογισμικού μέσω ενός συνδυασμού από προγραμματισμό (γραμμές
-κώδικα), δοκιμές επαλήθευσης, δοκιμές μονάδων, δοκιμές ολοκλήρωσης
-και αποσφαλμάτωση.
-Το γνωστικό πεδίο της κατασκευής λογισμικού συνδέεται και με πεδία
-γνώσης της πληροφορικής, ωστόσο είναι άρρηκτα συνδεδεμένο με τη σχεδίαση
-λογισμικού και τον έλεγχο λογισμικού, διότι η διαδικασία της κατασκευής
-λογισμικού περιλαμβάνει σημαντικά ποσά σχεδίασης και ελέγχου λογισμικού.
-Η προαναφερθείσα διαδικασία χρησιμοποιεί τα αποτελέσματα της σχεδίασης
-και παρέχει εισροές στον έλεγχο (η σχεδίαση και ο έλεγχος σε αυτή
-την περίπτωση αναφέρονται στις δραστηριότητες, όχι στα πεδία γνώσης).
-Τα όρια μεταξύ της σχεδίασης, της κατασκευής και του ελέγχου (αν
-υπάρχουν) ποικίλλουν, καθώς εξαρτώνται από τις διεργασίες του κύκλου
-ζωής του λογισμικού που χρησιμοποιείται σε ένα έργο.
+λειτουργικού λογισμικού συνδυάζοντας κώδικα, επαλήθευση, δοκιμές ελέγχου, 
+δοκιμές μονάδων, δοκιμές ολοκλήρωσης και αποσφαλμάτωση.
+
+Το γνωστικό πεδίο της κατασκευής λογισμικού συνδέεται με πεδία
+γνώσης της πληροφορικής, αλλά είναι άρρηκτα συνδεδεμένο και με τη σχεδίαση
+και τον έλεγχο λογισμικού, διότι η κατασκευή εμπλέκει άμεσα θέματα σχεδίασης και ελέγχου. 
+Η προαναφερθείσα διαδικασία κατασκευής χρησιμοποιεί τα αποτελέσματα της σχεδίασης και παρέχει εισροές στον έλεγχο 
+(η σχεδίαση και ο έλεγχος σε αυτή την περίπτωση αναφέρονται στις δραστηριότητες, όχι στα πεδία γνώσης). 
+Τα όρια μεταξύ σχεδίασης, κατασκευής και ελέγχου (αν υπάρχουν) ποικίλλουν, 
+καθώς εξαρτώνται από τις διεργασίες του κύκλου ζωής του λογισμικού που χρησιμοποιείται σε ένα έργο.
+
 Αν και είναι πιθανό να εφαρμοστεί λεπτομερή σχεδίαση πριν το στάδιο
```

Comment:
much design work is performed during construction== πολύ σχεδιαστική δουλειά πραγματοποιείται κατά τη διάρκεια της κατασκευής != κατά τη διάρκεια της κατασκευής δίνεται μεγάλη έμφαση στο κομμάτι της σχεδίασης.

Εδώ γίνεται paraphrase και όχι μετάφραση!

Επίσης μην αφήνεις κενές γραμμές όπου δεν υπάρχουν